packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_ETH.sol

```diff
@@ -18,17 +12,18 @@ import { OVM_L2DepositedERC20 } from "../bridge/tokens/OVM_L2DepositedERC20.sol"
  * Compiler used: optimistic-solc
  * Runtime target: OVM
  */
-contract OVM_ETH is OVM_L2DepositedERC20 {
-    constructor(
-        address _l2CrossDomainMessenger,
-        address _l1ETHGateway
-    )
-        OVM_L2DepositedERC20(
-            _l2CrossDomainMessenger,
+contract OVM_ETH is L2StandardERC20 {
+
+    /***************
+     * Constructor *
+     ***************/
+
+    constructor(address _l2Bridge)
```

Comment:
Nit: could just be a constant using `Lib_PredeployAddresses`. Would require changing deploy config in state dump process.